src/ui/components/Library/Team/View/TestRuns/hooks/useTestRuns.ts

```diff
@@ -11,33 +11,24 @@ import { testRunsIntervalCache } from "../suspense/TestRunsCache";
 
 const EMPTY_ARRAY: any[] = [];
 
-export function useTestRuns(): { testRuns: TestRun[]; status: Status } {
+export function useTestRuns(): { testRuns: TestRun[] } {
```

Comment:
A pattern we've been using in FE has been to name things that suspend explicitly. This makes it clearer that you need to use a Suspense boundary around them, or pass them deferred/transition priority values.

```suggestion
export function useTestRunsSuspends(): { testRuns: TestRun[] } {
```

Reply:
I like this :+1: